app/api/connectors/BaseDownstreamConnector.scala

```diff
@@ -33,6 +33,8 @@ trait BaseDownstreamConnector {
 
   private val jsonContentTypeHeader = HeaderNames.CONTENT_TYPE -> MimeTypes.JSON
 
+  implicit protected lazy val featureSwitches: FeatureSwitches = FeatureSwitches(appConfig.featureSwitches)
```

Comment:
This does not need to be implicit:
```suggestion
 protected lazy val featureSwitches: FeatureSwitches = FeatureSwitches(appConfig.featureSwitches)
```